### PR TITLE
Adding new pkg module for nova-rewrite

### PR DESCRIPTION
**Pull request by goravsingal**
Adding new pkg module for nova-rewrite

### CODE REVIEW

File: hubblestack/extmods/nova_v2_modules/pkg.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+Note: In this case it might return multiple packages, and if you have given version comparison too.
+All matched packages must comply to that version string.
+
+You can give multiple packages in the yaml, under "checks" attribute.
+"""
+import logging
+import os
+import re
+
+from distutils.version import LooseVersion
```
Comment (devagupt) [inline hubblestack/extmods/nova_v2_modules/pkg.py:39]: Loose version is outdated and will be deprecated in near future.
Please use version from packaging module.

File: hubblestack/extmods/nova_v2_modules/pkg.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+from hubblestack.utils.hubble_error import AuditCheckValdiationError
```
Comment (devagupt) [inline hubblestack/extmods/nova_v2_modules/pkg.py:41]: Please fix typo in exception name

File: hubblestack/extmods/nova_v2_modules/pkg.py (hunk 1 above)
Comment (devagupt) [inline hubblestack/extmods/nova_v2_modules/pkg.py]: Loose version is outdated and will be deprecated in near future.
Please use version from packaging module.

File: hubblestack/extmods/nova_v2_modules/pkg.py (hunk 2 above)
Comment (devagupt) [inline hubblestack/extmods/nova_v2_modules/pkg.py]: Please fix typo in exception name

**Review by devagupt — CHANGES_REQUESTED** [review]

File: hubblestack/extmods/nova_v2_modules/pkg.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+    if 'version' in audit_check:
+        pkg_version = audit_check['version'].replace(' ', '')
+        for package in pkg_found:
+            if not _match_version(pkg_version, pkg_found[package]):
+                errors[package] = "Package: %s, version found: %s, Expected: %s" %(package, pkg_found[package], pkg_version)
+
+    if errors:
+        # error string will have version along with operator if passed from profile
+        print("Package version mismatch: %s" %(str(errors)))
```
Comment (devagupt) [inline hubblestack/extmods/nova_v2_modules/pkg.py]: Remove this print statement

File: hubblestack/extmods/nova_v2_modules/pkg.py (hunk 1 above)
Comment (goravsingal) [reply hubblestack/extmods/nova_v2_modules/pkg.py]: done

File: hubblestack/extmods/nova_v2_modules/pkg.py (hunk 2 above)
Comment (goravsingal) [reply hubblestack/extmods/nova_v2_modules/pkg.py]: This was in original file too. Correcting at all places.

File: hubblestack/extmods/nova_v2_modules/pkg.py (hunk 3 above)
Comment (goravsingal) [reply hubblestack/extmods/nova_v2_modules/pkg.py]: done

**Review by devagupt — APPROVED** [review]

**Pull request merged**